tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,11 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    # raise Exception("Complete this test according to comments below.")
+    #Assert 
+    assert result is False
+    assert item_b in fatimah.inventory
+    assert nobodys_item not in jolie.inventory
+    assert jolie.inventory == []
+    assert fatimah.inventory == [item_a, item_b, item_c]
```

Comment:
We could remove the assertion on line 138. If we check that `jolie.inventory` is an empty list, then we don't need to check if `nobodys_item` is in `jolie.inventory` - It can't be present if the list is empty.

Similarly, we could remove the assertion on line 137, since below we check all the contents of `fatimah.inventory`.